HOSTING.md

```diff
@@ -33,5 +33,52 @@ To deploy on Render, please follow these steps:
 
 Then click "Save Changes", and your application should now be deployed.
 
+### Deploy vectara-answer on Render Using Render Create Service API
 
+Render provides the APIs to manage the services.
+
+Follow these steps to deploy Vectara Answer with the Render API:
+
+1. Create API Key: After logging in go to Account Settings and create the API Key. Checkout the [Render Docs](https://render.com/docs/api) for more information.
```

Comment:
"After logging in go to Account Settings and create the API Key." -> "Login to your account, go to 'Account Settings' and create an API Key."